python/mp/rosary.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, ip="127.0.0.1", port=5005, dispatcher=None, name="rosary"):
         self.beads = []
         self.bgcolor = color.Color(0,0,0)
         self.effects = []
-        self.triggers = {}
+        self.triggers = []
```

Comment:
Triggers is now treated like effects, a list of temporary instances rather than a permanent one